CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
```

Comment:
Is this a full copy from https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md? If so I'll skip readding it :-)

Reply:
Yes, its a copy and paste minus the support email